### PR TITLE
Cleanup create-react-app from unnecessary code/files and add file str…

### LINKED ISSUE #26: Cleanup create-react-app from unnecessary code/files and add file structure to the project.
add folders in /src.

- components
- pages
- themes

### PR DESCRIPTION
**Pull request by panchepanevski**
closes #26

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -2,14 +2,12 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
```
Comment (manuelwecker) [inline public/index.html]: You can use the fav icon and replace it later on with your logo

File: public/index.html [hunk 2]
```diff
@@ -2,14 +2,12 @@
...
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
     <meta
       name="description"
       content="Web site created using create-react-app"
     />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
```
Comment (manuelwecker) [inline public/index.html]: See above you can use the code line

File: public/manifest.json [hunk 3]
```diff
@@ -1,23 +1,7 @@
 {
   "short_name": "React App",
   "name": "Create React App Sample",
-  "icons": [
```
Comment (manuelwecker) [inline public/manifest.json]: See above, you can use the icons later

File: public/index.html (hunk 2 above)
Comment (golubs) [inline public/index.html]: You could leave this in for now, or you should remember to replace these with your own favicon and logo later, if you do want to delete them now.

File: src/index.js [hunk 4]
```diff
@@ -1,12 +1,8 @@
...
+import ReactDOM from "react-dom";
+import App from "./App";
+import * as serviceWorker from "./serviceWorker";
 
-ReactDOM.render(<App />, document.getElementById('root'));
+ReactDOM.render(<App />, document.getElementById("root"));
 
-// If you want your app to work offline and load faster, you can change
-// unregister() to register() below. Note this comes with some pitfalls.
-// Learn more about service workers: https://bit.ly/CRA-PWA
```
Comment (golubs) [inline src/index.js]: Why did you remove this comment? It's a good comment

File: .gitignore [hunk 5]
```diff
@@ -5,6 +5,7 @@
 /.pnp
 .pnp.js
 
+
```
Comment (holgermanzke) [inline .gitignore]: not necessary.

File: public/manifest.json [hunk 6]
```diff
@@ -1,23 +1,7 @@
...
-      "src": "logo192.png",
-      "type": "image/png",
-      "sizes": "192x192"
-    },
-    {
-      "src": "logo512.png",
-      "type": "image/png",
-      "sizes": "512x512"
-    }
-  ],
```
Comment (golubs) [inline public/manifest.json]: This is good code. You should just replace the icons with your own with the same names.

File: src/index.js (hunk 4 above)
Comment (tobiaserich) [reply src/index.js]: But not necessary for his project @golubs
